nin/backend/fontgen.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs-promise');
+const glob = require('glob');
+const path = require('path');
+
+
+async function fontGen(pathPrefix) {
+
+  let resolver;
+  let promise = new Promise(resolve => {
```

Comment:
I agree it's nice to reduce rightwards drift, although i find having everything inside the promise to be more readable.

Piece by piece fleshing out the Ninjadev style guide 💃 @stianjensen ?